Docs: Agregar resumen del capitulo 6

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 6 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Docs: Agregar resumen del capitulo 6

## what

- Agregar el resumen del capitulo 6 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch6/Ch6.md:
```diff
@@ -0,0 +1,161 @@
+
+Versiones anteriores a la 9 no tiene esta API, pero si configuras bien tu `fallback`, este debería cubrir estas situaciones, ademas de marcas de celulares más pequeñas cuentan con sus API´s únicas diferentes a la de la W3C.
+
+## Geoposition,js
```
creo que esta es una coma y debe ser un punto.

semanticmx commented on Ch6/Ejemplo/index.html:
```diff
@@ -0,0 +1,24 @@
+        <title>Geolocalización</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="geoPosition.js"></script>
```
en el ejemplo de arriba el script lo colocaste como parte del body.

Hay ventajas y desventajas en colocarlo en el body o en el head.

Si se coloca en el `<head>` dado que HTML es un lenguaje interpretado la carga del sitio se bloqueará por unos instantes y en ese tiempo verás una página en blanco, hasta que termine la ejecución del script. Esto no es deseable y se puntua bajo en los scripts que analizan el performance de las páginas.

https://www.modpagespeed.com/doc/filter-js-defer

Como se menciona en el link de arriba, lo recomendable es esperar a que todo el HTML se haya cargado para hasta ese momento cargar scripts, esto se logra de varias formas, la más fácil siendo agregar la etiqueta <script> justo antes de la etiqueta de cierre </body>

Solo que antes de usar la librería debemos esperar a que haya sucedido el evento onload, que se ejecuta una vez que todo el HTML de la página ha sido cargado.

Para este ejemplo dado que dentro de main.js solo hay funciones que se ejecutan al dar click en un botón es seguro mover estos dos scripts al final de la página.

Como lo estas corriendo de forma local tal vez no notes la diferencia pero por eso existen herramientas que detectan este tipo de ajustes y calculan el impacto en el tiempo de carga de las páginas.

semanticmx reviewed (changes requested):
muy bien!

IIKUYY committed:
Chores: realizar cambios necesarios

## what

- Corregir error de ortografía
- Mejorar el código moviendo los <scripts> a la parte final del código

## why

- Arreglar la semántica de los archivos
- Mejorar la optimización del código

## references
-

IIKUYY commented:
Se realizaron los cambios necesarios

semanticmx reviewed (approved):
perfecto!
